Gems/ImGui/Code/Source/ImGuiManager.cpp

```diff
@@ -452,6 +454,21 @@ bool ImGuiManager::OnInputChannelEventFiltered(const InputChannel& inputChannel)
             {
                 ToggleThroughImGuiVisibleState();
             }
+
+            if (inputChannel.GetInputChannelId() == AzFramework::InputDeviceKeyboard::Key::Escape)
+            {
+                CSystem *cSys = (CSystem*)GetISystem();
```

Comment:
standards: static_cast should be preferred here instead of a C-style cast according to the coding standards: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#casting

Reply:
Fixed in 368d2201a0598928e0e9ee5b1fed65de4fd55658